src/helpers/LatexHelpers.ts

```diff
@@ -0,0 +1,23 @@
+export const Santinize = (message: string): string => {
+    let cleanedMessage = message
+    let res_commands = [
+        [`\\text{`, `\\backslash text~{`],
+        ['$', `\\$`],
+        ['"', '\\"']
+    ]
+
+    let regex = /\\\\/g;
+    let result = cleanedMessage.replace(regex, "\\");
+    res_commands.forEach(([res, replace]) => {
+        result = result.replace(res, replace)
+    })
+    return cleanedMessage
+
+}
+
+import katex from 'katex';
+import * as htmlToImage from 'html-to-image';
+
+export const EquationRender = (message: string) => {
+    
```

Comment:
I will move the whole implementation in equation.ts here if it works.

Reply:
I disagree with leaving things until "if it works". You should be able to test your code locally to know if it works or not.